vulns/idna/PYSEC-2024-60.yaml

```diff
@@ -23,7 +23,8 @@ affected:
     repo: https://github.com/kjd/idna
   - type: ECOSYSTEM
     events:
-    - introduced: "0"
+    - introduced: "0.1"
+      fixed: "3.7"
```

Comment:
Without a hyphen in front of this, it doesn't wind up being a valid OSV record (both the `introduced` and `fixed` versions wind up being in the same `event` array entry object). Compare and contrast with the `GIT` range.

Reply:
https://github.com/pypa/advisory-database/pull/187